scripts/globals/magic.lua

```diff
@@ -336,7 +336,7 @@ function applyResistance(player,spell,target,diff,skill,bonus)
     end
 
     --get the base acc (just skill plus magic acc mod)
-    local magicacc = player:getSkillLevel(skill) + player:getMod(79 + skill) + player:getMod(MOD_MACC) + player:getILvlMacc();
+    local magicacc = player:getILvlMacc();

```

Comment:
Item level magic acc should be in the same place as the skill level macc (it is basically the same thing)